app/src/test/java/nl/tudelft/b_b_w/model/DatabaseHandlerUnitTest.java

```diff
@@ -42,7 +42,7 @@
     private final String iban = "iban";
     private final String publicKey = "publicKey";
     private Block _block;
-    private final int trustValue = 0;
+    private final int trustValue = TrustValues.INITIALIZED.getValue();
```

Comment:
get methods are inlined by any compiler > 2003; they do not add runtime complexity